components/cli/pkg/commands/delete_image.go

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2018 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http:www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package commands
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+
+	"github.com/cellery-io/sdk/components/cli/pkg/constants"
+	"github.com/cellery-io/sdk/components/cli/pkg/util"
+)
+
+func RunDeleteImage(images string) {
+	imagesToBeDeleted := strings.Split(images, ",")
+	// Get the list of images in repo
+	existingImages := getImagesArray()
+	for _, imageToBeDeleted := range imagesToBeDeleted {
+		imageExists := false
+		for _, existingImage := range existingImages {
+			var err error
+			imageExists, err = regexp.MatchString(fmt.Sprintf("^%s$", imageToBeDeleted), existingImage.name)
```

Comment:
Is there any reason to add `^%s$`?

Reply:
The reason to add "^%s$" is to check if a full string matches.
* The caret ^ matches the beginning of a text or line.
* The dollar sign $ matches the end of a text.

If "^%s$" was not used cell images which the user did not intend to delete might get deleted.
Eg: There are 2 cell images 
       1. wso2/sample:1.0.0
       2. testwso2/sample:1.0.0
Suppose the user entered following command to delete all images of wso2 organization.
`cellery delete 'wso2/.*'`
Without "^%s$" both images would get deleted. This can be avoided by using "^%s$".

Reply:
Users will anyway add `^` and `$` if required as we are telling them to use regex. We have to either document this addition to avoid confusion or remove adding  `^` and `$` so that user has the full control.

Reply:
Updated and removed `^` and `$` from regex to give full control to user.